lua/swenv/init.lua

```diff
@@ -8,6 +8,7 @@ end
 
 M.setup = function(opts)
   update_config(opts)
+  require('swenv.api').init()
```

Comment:
[question] should this be optional? previously the `api` module was not loaded on startup with `plenary`. not sure if it's important though

Reply:
umm sure, how about invoking it only when the filetype is python?

Reply:
umm I'm not quite sure I understand what you mean. But I notice that we could only invoke it iff the filetype is python. what do you think?

Reply:
Yeah nevermind actually :) The user can just choose to call setup later if they want :+1: